code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1336,4 +1336,151 @@
 	. = 1
 
 
+/datum/reagent/medicine/burnmix
+	name = "BurnMix"
+	description = "Take in moderation. Heals damage at a snail's pace. Effects increase when used with Epinephrine or Perfluorodecalin. You don't want this in your body for longer than needed."
+	reagent_state = LIQUID
+	color = "#BD328A" //  R 189 G 50 B 138 , Medium Red Violet , Hue of Violet 
+	overdose_threshold = 12
+	taste_description = "dixel"
+	glass_name = "glass of ...red?"
+	glass_desc = "Looks like some chunky dark red mix."
+	reagent_weight = 4
+	addiction_threshold = 40
```

Comment:
Hello Alex,
The whole point of BurnMix is to be able to enter the OD probability effects. "OD" for BurnMix isn't necessary a "bad" thing. 

Do you mean that the chemistry variant should be like a "purified" version not able to get in to the OD effects? I would be open to this but I'd have to test the balance of this since in my mind currently it would make BurnMix not viable if made inside of Chemistry.

Reply:
Ohh I meant addiction not overdose, sorry bout that

Reply:
Good suggestion. Will make a snowflake friendly "purified" variant in another PR.